sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -3,12 +3,19 @@
 ## 4.1.0-beta.3 (Unreleased)
```

Comment:
Update version?

Reply:
Will create a separate PR for version updates/